src/cata_tiles.h

```diff
@@ -3,6 +3,7 @@
 
 #include <SDL.h>
 #include <SDL_ttf.h>
+#include <SDL/SDL_gpu.h>
```

Comment:
This is a reasonable change, but it will need some work on other platforms to work correctly (ideally, pkg-config support upstream, I'll see what I can do)